change/@microsoft-adaptive-ui-c42f4c4e-b1c3-464a-93fe-fa308986cdd4.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Fix a color bug in Swatch",
```

Comment:
Could this be more descriptive on what is being fixed? That will make it easier for folks who come looking to find a clear resolution in the changelog